addons/web/static/lib/bootstrap/scss/_variables.scss

```diff
@@ -867,7 +867,7 @@ $form-feedback-tooltip-border-radius: $tooltip-border-radius !default;
 
 $popover-font-size:                 $font-size-sm !default;
 $popover-bg:                        $white !default;
-$popover-max-width:                 276px !default;
+$popover-max-width:                 552px !default;
```

Comment:
Changing the size of all the popover of the website in stable for a forum fix seems a bad idea. Could not this be only about changing that particular popover problem?

Reply:
It seems that the `popover-max-width` is set for the `.popover` element which is created by the library, and I have access only to the child of this element.

Reply:
In JS you have access to everything. There are already places where we add a class on a specific popover to be able to change its width specifically

Reply:
if it can help https://github.com/odoo/odoo/pull/64756/commits/6079d36034f51bcde3bac7b3dec71d4ee40b0615#diff-70f7fe38208aa7fe678f18e329d3c11b70065dee723921352b6005774e8bab53R1108

Reply:
:+1:

Reply:
What is the best way to do it: the one proposed by @rdeodoo or this one https://github.com/odoo/odoo/blob/d9b9c4e8e11bb356f97b7917d48bed3eaf988af2/addons/website_sale/static/src/js/website_sale.js#L19

Reply:
Whatever is easier for your case. I like RDE's one since it does not require to know the full structure of the tooltip... we just want to add a class here.